OWNERS

```diff
@@ -1,43 +1,32 @@
 approvers:
   - Luap99
   - TomSweeneyRedHat
-  - alexlarsson
   - ashley-cui
   - baude
   - edsantiago
-  - flouthoc
   - giuseppe
-  - jakecorrenti
-  - jnovy
-  - jwhonce
+  - l0rd
   - lsm5
   - mheon
   - mtrmac
-  - n1hility
   - rhatdan
-  - saschagrunert
-  - umohnani8
   - vrothberg
-  - ygalblum
 reviewers:
```

Comment:
If approvers and reviewers should be different can we define once a) What is the difference? And b) what is the criteria to qualify as reviewer but not approver then?

Reply:
ofc we can ... but for now, this is just a quick audit of things